libcodechecker/libhandlers/analyze.py

```diff
@@ -174,6 +174,14 @@ def is_ctu_capable():
                                     "compiler on the system is special, e.g. "
                                     "when doing cross-compilation.")
 
+    analyzer_opts.add_argument('--capture-analysis-output',
```

Comment:
A'ight, as discussed, every "ideal" optstring would be too long. And just for the record, I'm noting that the suggestion about turning fail-zipping off was dropped.